data/data/bootstrap/files/usr/local/bin/approve-csr.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+KUBECONFIG="${1}"
+
+echo "Approving all CSR requests until bootstrapping is complete..."
+while [ ! -f /opt/openshift/bootkube.done ]
+    oc --config="$KUBECONFIG" get csr --no-headers | grep Pending | \
+        awk '{print $1}' | \
+        xargs --no-run-if-empty oc --config="$KUBECONFIG" adm certificate approve
+do
```

Comment:
Hm, does bash support multiple lines for `while` conditions?  I hadn't noticed that before.

Reply:
That's not part of the condition, rather the body.

Reply:
Hm, is there any meaning to having it *before* the `do`?

Reply:
Oh, my bad. fixed.